crates/matrix-sdk-indexeddb/src/state_store.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::{collections::BTreeSet, sync::Arc};
+use std::{collections::BTreeSet, path::PathBuf, sync::Arc};
```

Comment:
Not sure if this is a good idea on the indexeddb store?

Reply:
yeah, no. this would probably build, but the entire _assumption_ that db has a `PathBuf` is just flawed: it makes the specific of one implementation the default for all. That's not how we should model this (PostgresSQL won't have one either, nor Redis ...)